src/Attribute/AsOption.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Castor\Attribute;
+
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
+class AsOption
+{
+    /**
+     * @param string|array<string>|null                $shortcut
+     * @param string|int|bool|array<string>|float|null $default
+     * @param array<string>                            $suggestedValues
+     */
+    public function __construct(
+        public string|null $name = null,
+        public string|array|null $shortcut = null,
+        public int|null $mode = null,
+        public string $description = '',
+        public string|int|bool|array|null|float $default = null,
```

Comment:
is this really useful ? i don't think it's a good approach (it's better to use the default value of the function declaration as it can be used when calling a function from another function)

Reply:
I don't even use it 🤦🏼. It's a reliqua of a previous version. I removed it! thanks